### DISCUSSION TITLE
The SH1mmer guide for dummies.

### DISCUSSION THREAD

**Discussion by FearedFusionX** [3kh0/ext-remover#652] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
[https://rentry.co/pots2x](https://rentry.co/pots2x)

thank me later

**Comment by Hilololsusy** [comment]
But... We already have a sh1mmer guide

**Reply by FearedFusionX** [reply]
True, but this one shows you how to downgrade to make SH1mmer actually work.

**Reply by FearedFusionX** [reply]
And the SH1mmer URL is blocked for schools

**Comment by Hilololsusy** [comment]
But... We already have a sh1mmer guide
